frontend/app-development/features/processEditor/ProcessEditor.tsx

```diff
@@ -23,6 +27,25 @@ export const ProcessEditor = (): React.ReactElement => {
   const existingCustomReceipt: string | undefined = useCustomReceiptLayoutSetName(org, app);
   const bpmnMutation = useBpmnMutation(org, app);
 
+  const { onWSMessageReceived } = useWebSocket({
+    webSocketUrl: processEditorWebSocketHub(),
+    webSocketConnector: WSConnector,
+  });
+
+  onWSMessageReceived<SyncError | SyncSuccess>((message): void => {
+    // Check if the message is a SyncError
```

Comment:
Are the comments added in this file necessary?

Reply:
hm, what do you think? I added the comment to ensure that other developers understand that 'errorCode' in message means that this is an error message.

But it may be better to refactor it:

```typescript
const isErrorMessage = 'errorCode' in message;
if (isErrorMessage) {
    toast.error(t(SyncUtils.getSyncErrorMessage(message)));
      return;
}
```

Reply:
I like the newly suggested code 😄